ada_feeding/ada_feeding/behaviors/ros/time.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This module defines utility behaviors related to ROS time.
+"""
+
+# Standard imports
+from typing import Optional
+
+# Third-party imports
+from overrides import override
+import py_trees
+
+# Local imports
+from ada_feeding.behaviors import BlackboardBehavior
+from ada_feeding.helpers import BlackboardKey
+
+
+class TrackHzInitialize(BlackboardBehavior):
```

Comment:
I would rename this as `WriteTimeToBlackboard`, since that's in effect what you're doing. And remove num_ticks.